account/do-account.js

```diff
@@ -5,7 +5,7 @@ class Account {
    * @param {object} doSdk - DigitalOcean SDK
    * @param {string} token - DigitalOcean token
    */
-  constructor(doSdk, token) {
+  constructor(doSdk,token) {
```

Comment:
Did you format with Prettier?

Reply:
Not really. I am casually very particular about syntax and indentations. So I keep looking for them.